flutter-idea/src/io/flutter/sdk/FlutterCommand.java

```diff
@@ -272,6 +272,7 @@ enum Type {
     CONFIG("Flutter config", "config"),
     CREATE("Flutter create", "create"),
     DOCTOR("Flutter doctor", "doctor", "--verbose"),
+    PUB_Add("Flutter pub add", "pub", "add"),
```

Comment:
I was a bit confused why this wasn't all caps so idk if that matters or if you know where its coming from

Reply:
Oh I just realized I typed this manually LOL, updating to match the others.